src/Infrastructure/LeanCode.Logging/IHostBuilderExtensions.cs

```diff
@@ -42,52 +43,58 @@ public static IHostBuilder ConfigureDefaultLogging(
+                    var loggerConfiguration = new LoggerConfiguration()
+                        .ReadFrom.Configuration(configuration)
+                        .Enrich.FromLogContext()
+                        .Enrich.WithProperty("project", projectName)
+                        .Enrich.WithProperty("app_name", appName)
+                        .MinimumLevel.Is(minLogLevel)
+                        .DestructureCommonObjects(destructurers);
 
-                if (!configuration.GetValue<bool>(EnableDetailedInternalLogsKey))
-                {
-                    var internalLogLevel =
-                        minLogLevel > InternalDefaultLogLevel ? minLogLevel : InternalDefaultLogLevel;
-                    loggerConfiguration
-                        .MinimumLevel.Override("Microsoft", internalLogLevel)
-                        .MinimumLevel.Override("System", internalLogLevel)
-                        .MinimumLevel.Override("Azure.Identity", internalLogLevel)
-                        .MinimumLevel.Override("Azure.Core", internalLogLevel);
-                }
+                    if (!configuration.GetValue<bool>(EnableDetailedInternalLogsKey))
+                    {
+                        var internalLogLevel =
+                            minLogLevel > InternalDefaultLogLevel ? minLogLevel : InternalDefaultLogLevel;
+                        loggerConfiguration
+                            .MinimumLevel.Override("Microsoft", internalLogLevel)
+                            .MinimumLevel.Override("System", internalLogLevel)
+                            .MinimumLevel.Override("Azure.Identity", internalLogLevel)
+                            .MinimumLevel.Override("Azure.Core", internalLogLevel);
+                    }
 
-                if (configuration.GetValue<string>(SeqEndpointKey) is string seqEndpoint)
-                {
-                    loggerConfiguration.WriteTo.Seq(seqEndpoint, formatProvider: CultureInfo.InvariantCulture);
-                }
+                    if (configuration.GetValue<string>(SeqEndpointKey) is string seqEndpoint)
+                    {
+                        loggerConfiguration.WriteTo.Seq(seqEndpoint, formatProvider: CultureInfo.InvariantCulture);
+                    }
 
-                if (context.HostingEnvironment.IsDevelopment())
-                {
-                    loggerConfiguration.WriteTo.Console(formatProvider: CultureInfo.InvariantCulture);
+                    if (context.HostingEnvironment.IsDevelopment())
+                    {
+                        loggerConfiguration.WriteTo.Console(formatProvider: CultureInfo.InvariantCulture);
+                    }
+                    else
+                    {
+                        loggerConfiguration.WriteTo.Console(new RenderedCompactJsonFormatter());
+                    }
+
+                    additionalLoggingConfiguration?.Invoke(context, loggerConfiguration);
+
+                    Log.Logger = loggerConfiguration.CreateLogger();
```

Comment:
This leaves the static logger available and we want to eliminate it completely.

Reply:
> I remember from corelib v8 that we wanted to minimize the number of breaking changes.
This is the kind of breaking change that I am willing to do. "Reverting" that on the app side is a matter of two-three lines.

Also, we can add `preserveStaticLogger` flag (defaulting to false), just like `AddSerilog` does if we want to do.